ui/app/document-upload/controllers/documentController.js

```diff
@@ -142,9 +143,21 @@ angular.module('opd.documentupload')
                         v: "custom:(uuid,setMembers:(uuid,name:(name)))"
                     }
                 }).then(function (response) {
+                    if (response.data.results[0].setMembers && response.data.results[0].setMembers.length > 0) {
+                        response.data.results[0].setMembers.forEach(function (concept) {
+                            var conceptToAdd = {
+                                'concept': {
+                                    uuid: concept.uuid,
+                                    name: concept.name.name,
+                                    editableName: concept.name.name
+                                }
+                            };
+                            $scope.allConcepts.push(conceptToAdd);
+                        });
+                    }
                     var topLevelConcept = response.data.results[0];
                     topLevelConceptUuid = topLevelConcept ? topLevelConcept.uuid : null;
-                    setDefaultConcept(topLevelConcept);
+                    // setDefaultConcept(topLevelConcept);
```

Comment:
- donot comment but remove the lines of old code to be removed as part this change
- if setDefaultConcept is not used, it can be removed.